go/filemanager/pfsmodules/status.go

```diff
@@ -0,0 +1,23 @@
+package pfsmodules
+
+// TODO
+// need a custom error type?
+
+const (
+	StatusFileNotFound          = "no such file or directory"
```

Comment:
After offline discussion with @gongweibao , it's used by the client to compare these string with the error message returned from the server over HTTP.